src/components/Calculator.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+const Calculator = () => {
+  const [state] = useState({
+    total: 0,
+  });
+  const { total } = state;
+  const disp = `${total}`;
+
+  return (
+    <div className="calc">
+      <span>{disp}</span>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="colored">÷</button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="colored">x</button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="colored">-</button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="colored">+</button>
+      <button type="button" className="double">0</button>
+      <button type="button">.</button>
+      <button type="button" className="colored">=</button>
+    </div>
+  );
+};
```

Comment:
- [ ] The Calculator component is expected to have at least 1 child component. What if you you could create a Button component and pass the button values to it as a child in the Calculator component? 👍🏽

<img width="913" alt="Screenshot 2023-03-25 at 2 44 03 PM" src="https://user-images.githubusercontent.com/50941074/227724559-a0eb4914-7599-4613-983f-e45adfb609ac.png">